plugins/scaffolder-backend/src/schema/openapi.yaml

```diff
@@ -0,0 +1,1319 @@
+        apiVersion:
+          type: string
+          description: |-
+            The version of specification format for this particular entity that
+            this is written against.
+      required:
+        - metadata
+        - kind
+        - apiVersion
+      description: The parts of the format that's common to all versions/kinds of entity.
+    Error:
+      type: object
+      properties:
+        error:
+          type: object
+          properties:
+            name:
+              type: string
+            message:
+              type: string
+            stack:
+              type: string
+            code:
+              type: string
+          required:
+            - name
+            - message
+        request:
+          type: object
+          properties:
+            method:
+              type: string
+            url:
+              type: string
+          required:
+            - method
+            - url
+        response:
+          type: object
+          properties:
+            statusCode:
+              type: number
+          required:
+            - statusCode
+      required:
+        - error
+        - response
+      additionalProperties: {}
+    JsonArray:
+      type: array
+      items:
+        $ref: '#/components/schemas/JsonValue'
+      description: A type representing all allowed JSON array values.
+    JsonObject:
+      type: object
+      properties: {}
+      description: A type representing all allowed JSON object values.
+      additionalProperties: {}
+    JsonPrimitive:
+      oneOf:
+        - type: boolean
+        - type: number
+        - type: string
+        - type: object
+          nullable: true
+      description: A type representing all allowed JSON primitive values.
+    JsonValue:
+      oneOf:
+        - $ref: '#/components/schemas/JsonObject'
+        - $ref: '#/components/schemas/JsonArray'
+        - $ref: '#/components/schemas/JsonPrimitive'
+      description: A type representing all allowed JSON values.
+    ListActionsResponse:
+      type: array
+      items:
+        $ref: '#/components/schemas/Action'
+      description: The response shape for the `listActions` call to the `scaffolder-backend`
+    ListTasksResponse:
+      type: object
+      properties:
+        tasks:
+          type: array
+          items:
+            $ref: '#/components/schemas/SerializedTask'
+        totalTasks:
+          type: number
+      required:
+        - tasks
+      description: The response shape for the `listTasks` call to the `scaffolder-backend`
+    MapStringString:
+      type: object
+      properties: {}
+      additionalProperties:
+        type: string
+      description: Construct a type with a set of properties K of type T
+    ScaffolderScaffoldOptions:
+      type: object
+      properties:
+        templateRef:
+          type: string
+        values:
+          type: object
+          additionalProperties:
+            $ref: '#/components/schemas/JsonValue'
```

Comment:
By default, all values of a payload should be JSON encode-able, we could probably just do `additionalProperties: true` here.